lib/layer/core.py

```diff
@@ -982,17 +977,16 @@ def restore_to_layer(self, layer):
         layer.locked = self.locked
 
 
-class ExternallyEditable:
+class ExternallyEditable(metaclass=abc.ABCMeta):
     """Interface for layers which can be edited in an external app"""
 
-    __metaclass__ = abc.ABCMeta
     _EDITS_SUBDIR = "edits"
 
     @abc.abstractmethod
     def new_external_edit_tempfile(self):
         """Get a tempfile for editing in an external app
 
-        :rtype: unicode/str
+        :rtype: str/str
```

Comment:
I think the slash was an either/or:
```suggestion
        :rtype: str
```